lib/widget/image.dart

```diff
@@ -86,27 +88,41 @@ class ImageState extends WidgetState<EnsembleImage> {
         // image binding is tricky. When the URL has not been resolved
         // the image will throw exception. We have to use a permanent placeholder
         // until the binding engages
-        return 
-        CachedNetworkImage(
-                  width: widget._controller.width?.toDouble(),
+        if (widget._controller.cache == true) {
```

Comment:
@Moeez1999 you can just do if (widget._controller.cache)
no need for == true

Reply:
sure we can do that